src/sage/plot/plot3d/plot_field3d.py

```diff
@@ -126,8 +126,8 @@ def plot_vector_field3d(functions, xrange, yrange, zrange,
     vectors = [vector((ff(*point), gg(*point), hh(*point))) for point in points]
 
     try:
-        from matplotlib.cm import get_cmap
-        cm = get_cmap(colors)
+        import matplotlib
+        cm = matplotlib.colormaps.get_cmap(colors)
     except (TypeError, ValueError):
```

Comment:
```suggestion
        import matplotlib as mpl
        cm = mpl.colormaps[colors]
    except (TypeError, KeyError):
```

Reply:
I will be careful with that one. If I am not mistaken `matplotlib` is used at least another time in this section and will need to be changed to `mpl`.

Reply:
I was mistaken, I was thinking of another section using `mpl` already.

Reply:
Either should work ok, I suggested changing it to `mpl` for consistency.